### ISSUE TITLE
With same trainging data, the prediction results are getting different

### ISSUE THREAD

**Issue by suresh-soundararajan** [facebookresearch/fastText#586]
Hi Team,

I am having a binary classification use case like to rebuild the model everytime using the same set of training data. For every build the loss value is getting huge difference. Each time the results are drastically deviating.

with the -lr 0.5 -epoch 50 -loss ns

Am I missing any parameter.

Your help is much appreciated.

Thanks,
SureshKumar S

**Comment by EdouardGrave** [comment]
Hi @suresh-soundararajan,

Do you also observe difference in accuracy when you evaluate the corresponding model? Could you please provide the different values of the loss you observe? And finally, what is the size of your dataset? Thanks for you help!

Best,
Edouard.
